terra_layer/style.py

```diff
@@ -489,27 +632,45 @@ def generate_style_from_wizard(layer, config):
             radius_base = ["sqrt", ["/", get_field_style(field), ["pi"]]]
             radius = gen_style_interpolate(radius_base, boundaries, sizes)
 
+            config_style = {
```

Comment:
Why don't you use update ?

Reply:
It is more concise to do like this, than clone, update and filter the dict.